consensus/ethash/consensus.go

```diff
@@ -799,3 +848,35 @@ func GetBlockEra(blockNum, eraLength *big.Int) *big.Int {
 
 	return new(big.Int).Sub(d, dremainder)
 }
+
+// callistoAccumulateRewards()
+func callistoAccumulateRewards(config *params.ChainConfig, state *state.StateDB, header *types.Header, uncles []*types.Header) {
+	// Select the correct block reward based on chain progression
+	blockReward := CLOMinerReward
+	clotreasury := CLOTreasuryReward
+	clostake := CLOStakeReward
+
+	// Accumulate the rewards for the miner and any included uncles
+	reward := new(big.Int).Set(blockReward)
+	r := new(big.Int)
+	for _, uncle := range uncles {
+		r.Add(uncle.Number, big8)
+		r.Sub(r, header.Number)
+		r.Mul(r, blockReward)
+		r.Div(r, big8)
+		state.AddBalance(uncle.Coinbase, r)
+
+		r.Div(blockReward, big32)
+		reward.Add(reward, r)
+	}
+	// Activate Callisto hardfork
+	if config.IsCLOHF1(header.Number) {
```

Comment:
we could set address in variables and after the `if` we could run the function `.AddBalance`

And the treasury and stake reward will change too after HF1

Total treasury fee at this stage is 30% which includes:
● Development funding: 10%
● Cold staking allocation: 20%

Reply:
@yograterol That's correct

Reply:
@akx20000a ok, but where do you calc the new cloTreasury and cloStake amount?